### DISCUSSION TITLE
Not able to make timelapses

### DISCUSSION THREAD

**Discussion by gigaklatch** [gtxaspec/wz_mini_hacks#597] [Q&A]
Hello,

Is anyone else having issues with the timelapse functionallity of these cameras? After having set the time period for the time lapse, it seems do be doing its job making the timelapse, but then at the end nothing ever shows up, the timelapse is not found anywhere.

Could the mini hacks be interfering with the timelapse-making? Something else that I have missed?

**Comment by vic4news** [comment]
I had that issue with older versions of the wyze firmware on my v3 cameras. Nothing to do with mini hacks. I now run wyze firmware 4.36.9.139 with mini hacks and Timelapse is working.

**Reply by gigaklatch** [reply]
Okay, I see. Unfortunately my three cameras are already on that same firmware.  Though, fortunately, I have had one of them succesfully produce a timelapse today, twice. No idea what I did do differently. This certainly warrants some more attention and perhaps intentionally trying to make the timelapse procedure brake. I need to know what I should avoid doing in order for them to reliably give me the timelapses I set.

**Comment by robughblah** [comment]
I haven’t tried it with mini hacks, but I generally only get time lapse to work 50% of the time with results as you described in your email. RobOn Aug 24, 2023, at 2:47 PM, gigaklatch ***@***.***> wrote:﻿
Okay, I see. Unfortunately my three cameras are already on that same firmware.  Though, fortunately, I have had one of them succesfully produce a timelapse today, twice. No idea what I did do differently. This certainly warrants some more attention and perhaps intentionally trying to make the timelapse procedure brake. I need to know what I should avoid doing in order for them to reliably give me the timelapses I set.
